### PR TITLE
[Enhance] Support MMCLI

### PR DESCRIPTION
**Pull request by ToumaKazusa3**
Support installing MMTracking with MMCLI.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+include mmtrack/VERSION
+include mmtrack/model_zoo.yml
+include mmtrack/configs/*/*.py
+include mmtrack/configs/*/*.yml
+include mmtrack/configs/*/*/*.py
+include mmtrack/configs/*/*/*.yml
+include mmtrack/tools/*.py
+include mmtrack/tools/*.sh
+include mmtrack/tools/*/*.py
+include mmtrack/demo/*
```
Comment (OceanPang) [inline MANIFEST.in:11]: 1. Use recursive-include to update the code, refer [here](https://github.com/open-mmlab/mmdetection/commit/69d4b4d485934c4264a0019f429ab26fb52bb118)
2. Add an empty line at the end of the file.

File: setup.py [hunk 2]
```diff
@@ -135,7 +135,8 @@ def gen_packages_items():
         author_email='openmmlab@gmail.com',
         keywords='computer vision, object tracking, video object detection',
         url='https://github.com/open-mmlab/mmtracking',
-        packages=find_packages(exclude=('configs', 'tools', 'demo')),
+        packages=find_packages(),
```
Comment (OceanPang) [inline setup.py:138]: Why do you modify here?

File: MANIFEST.in (hunk 1 above)
Comment (ToumaKazusa3) [reply MANIFEST.in]: I have fixed these two issues.

File: setup.py (hunk 2 above)
Comment (ToumaKazusa3) [reply setup.py]: I have modified the code back.

**Review by OceanPang — APPROVED** [review]

**Pull request merged**